test/conftest.py

```diff
@@ -167,9 +177,9 @@ async def fails_without_raft(request, check_pre_raft):
 
 # "keyspace" fixture: Creates and returns a temporary keyspace to be
 # used in tests that need a keyspace. The keyspace is created with RF=1,
-# and automatically deleted at the end. We use scope="session" so that all
-# tests will reuse the same keyspace.
-@pytest.fixture(scope="session")
+# and automatically deleted at the end.
+@pytest.mark.asyncio
+@pytest.fixture(scope="function")
```

Comment:
This change is even bigger. It means a new keyspace is created for every test.
Again, I'm fine with this is that's what you want, but as long as it doesn't read cql-pytest.

Reply:
Please note that because each keyspace will have a unique name, this will suffer from the still-existing bug that Scylla has when you create and delete a lot of keyspaces or tables with different names.

Reply:
A new cluster is created for almost every topology test case. Same reasoning as driver connection.